services/user.service.js

```diff
@@ -0,0 +1,16 @@
+import { User } from '../db/models/user.js';
+import { EntityService } from './entity.service.js';
+
+export class UserService extends EntityService {
+  constructor() {
+    super(User);
+  }
+
+  async getById(id) {
```

Comment:
You can use just getOne of entityService or add separate method findById to entityService.